apps/nextjs-website/src/components/organisms/WebinarQuestionsForm/WebinarQuestionsForm.tsx

```diff
@@ -1,39 +1,62 @@
+'use client';
+import { snackbarAutoHideDurationMs } from '@/config';
+import { addWebinarQuestion } from '@/helpers/webinarQuestions.helpers';
+import { DevPortalUser } from '@/lib/types/auth';
 import { Done } from '@mui/icons-material';
 import { LoadingButton } from '@mui/lab';
-import { Card, TextField, Typography } from '@mui/material';
+import { Alert, Card, Snackbar, TextField, Typography } from '@mui/material';
 import { useTranslations } from 'next-intl';
-import { useState } from 'react';
-
-const sendQuestion = (question: string) => {
-  return new Promise((resolve) => {
-    setTimeout(() => {
-      resolve('ok');
-    }, 3000);
-  });
-};
+import { useCallback, useState } from 'react';
 
 type FormState = 'submitting' | 'submitted' | undefined;
 type WebinarQuestionsFormProps = {
   disabled?: boolean;
+  webinarSlug: string;
+  user: DevPortalUser;
 };
 
 export const WebinarQuestionsForm = ({
   disabled = false,
+  webinarSlug,
+  user,
 }: WebinarQuestionsFormProps) => {
   const t = useTranslations('webinar');
+  const [error, setError] = useState<string | null>(null);
   const [question, setQuestion] = useState('');
   const [formState, setFormState] = useState<FormState>();
 
+  const sendQuestion = useCallback(async () => {
+    return await addWebinarQuestion({
+      email: user.attributes.email,
+      given_name: user.attributes.given_name,
+      family_name: user.attributes.family_name,
+      question: question,
+      webinarSlug: webinarSlug,
+      data: new Date().toISOString(),
+    });
+  }, [webinarSlug, user, question]);
```

Comment:
This is related to [this comment](https://github.com/pagopa/developer-portal/pull/500/files#r1422557636): I'd use camelCase for the attributes name (e.g. `given_name` should become `givenName` and so on